server.js

```diff
@@ -5,21 +5,18 @@ const app = require('./src/app');
 const env = nconf.get('env');
 
 // Check ES (if in use)
-if (env.ES_HOST && typeof env.ES_HOST === 'string') {
+if (env.ES_HOST) {
   const runES = require('./src/run-es');
   const checkES = new runES.Elastic();
-  checkES.setup()
-    .then(() => {
-      startListen();
-    });
+  checkES.setup();
 } else {
-  startListen();
-  logger.debug(`No (valid) Elasticsearch host provided - results will NOT be saved!`);
+  logger.debug(`[Elasticsearch] no host provided in config [${env.APP_CONFIG}] - results will NOT be saved!`);
 }
+startListen();
```

Comment:
`startListen` no longer waits for `checkES.setup()` to run - is this intentional?

Reply:
Yes - i decided to start the API asap since it also serves swagger docs, etc.. The API won't perform any ES functions until `checkES.setup()` has completed successfully and it will inform clients of the status in the JSON responses